lib/scan/subdomain/engine.py

```diff
@@ -16,7 +16,7 @@
 from lib.socks_resolver.engine import getaddrinfo
 from core._time import now
 from core.log import __log_into_file
-import censys.certificates
+import censys
```

Comment:
@itsdivyanshjain also import:
from censys.search import CensysCertificates

Reply:
From This line: https://github.com/OWASP/Nettacker/pull/430/files/034457ce245c0e4bf2e2308327fc7ea7d2aa79f0#diff-068f0006f6329304a13e6d1e96292b7fffee3a5958301d5901194826a92a4bccR758 to this line: https://github.com/OWASP/Nettacker/pull/430/files/034457ce245c0e4bf2e2308327fc7ea7d2aa79f0#diff-068f0006f6329304a13e6d1e96292b7fffee3a5958301d5901194826a92a4bccR761

Please change the code to the given below:
```
try:
    cen_certificates = CensysCertificates(censys_api_key[0], censys_secret[0])
except censys.base.CensysUnauthorizedException:
    return []
except censys.base.CensysRateLimitExceededException:
    return []
subs = []
query = "parsed.names: {}".format(target)
search_results = cen_certificates.search(query, fields=["parsed.names"])
for i in search_results:
    for j in list(i.values()):
        for k in j:
            if "*" not in k and k.endswith(target):
                subs.append(k)
```